tsconfig.json

```diff
@@ -3,6 +3,8 @@
     "target": "es5",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
+    "baseUrl": ".",
+    "downlevelIteration": true,
```

Comment:
this was necessary for the new hook. 
TypeScript wasn't having URLQueryParams be iterated.